bears/__init__.py

```diff
@@ -1,6 +1,11 @@
 import os
 import sys
 
+__doc__ = """
+          Coala-bears is a Python package containing all the bears that are officially supported by coala.
+          It features more than 78 bears covering 54 languages.
```

Comment:
About this, it's generally a good idea to stay away from hard numbers like this. This gives a responsibility to maintain these in the future forever. Maybe just link to https://github.com/coala/bear-docs so that they can find out for themselves?